docs/core/extensions/configuration.md

```diff
@@ -32,7 +32,36 @@ Given one or more configuration sources, the <xref:Microsoft.Extensions.Configur
 
 ## Configure console apps
 
-.NET console applications created using the [dotnet new](../tools/dotnet-new.md) command template or Visual Studio by default *do not* expose configuration capabilities. To add configuration in a new .NET console application, [add a package reference](../tools/dotnet-add-package.md) to [Microsoft.Extensions.Hosting](https://www.nuget.org/packages/Microsoft.Extensions.Hosting). Modify the *Program.cs* file to match the following code:
+.NET console applications created using the [dotnet new](../tools/dotnet-new.md) command template or Visual Studio by default *don't* expose configuration capabilities. To add configuration in a new .NET console application, [add a reference](../tools/dotnet-add-package.md) to [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration). This package is the foundation for configuration in .NET apps. It provides the <xref:Microsoft.Extensions.Configuration.ConfigurationBuilder> and related types.
+
+```csharp
+using Microsoft.Extensions.Configuration;
+
+var configuration = new ConfigurationBuilder()
+    .AddInMemoryCollection(new Dictionary<string, string?>()
+    {
+        ["SomeKey"] = "SomeValue"
+    })
+    .Build();
+
+Console.WriteLine(configuration["SomeKey"]);
+
+// Outputs:
+//   SomeValue
+```
```

Comment:
Why no snippet?

EDIT: Further, I think this example is kind of useless, sorry! 😁  If it's for configuration I would assume the most common scenario would be consuming the _appsettings.json_ file. I took this from my [modernizing a .net framework app after porting to .net article](https://learn.microsoft.com/en-us/dotnet/core/porting/modernize).

```csharp
using Microsoft.Extensions.Configuration;

internal class Program
{
    internal static IConfiguration Config { get; private set; }

    private static void Main(string[] args)
    {
        Config = new ConfigurationBuilder()
            .AddJsonFile("appsettings.json")
            .Build();

        // Use the config file to get a connection string
        string? myConnectionString = Config.GetConnectionString("database");

        // Run the rest of your app
    }
}
```

Then you could also show the appsettings.json file too.

Reply:
This makes sense in the context (and flow) of the article. I'll add it as a snippet.